requirements/main.in

```diff
@@ -0,0 +1,20 @@
+# Common dependencies
+pyyaml >= 5.1
+astropy >= 5.2
+sqlalchemy >= 1.4
+pydantic >=2,<3.0
+deprecated >=1.2
+pyarrow >= 0.16
+pandas >= 1.0
```

Comment:
pandas and numpy are not real butler dependencies. They are there for the tests like matplotlib. It's true that we have to make sure that the server can deal with storage classes that can't be expanded but I hope that does work already.